check-environment-and-install

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+return_code=0
+if [[ -n "${BROWSER_DRIVER_INSTALLER_CHROME_VERSION:-}" && -n "${BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH:-}" ]]; then
+    npx browserDriverInstaller --browser-name Chrome                                                \
+                    --browser-version "$BROWSER_DRIVER_INSTALLER_CHROME_VERSION"                    \
+                    --target-path "$BROWSER_DRIVER_INSTALLER_CHROMEDRIVER_PATH" || return_code=$?
+else
+    echo "The environment variables for Chrome are not set, skipping post-install for Chrome"
```

Comment:
list the environment variables themselves

Reply:
_the_ post-install _step_

Reply:
listed the env var names, changed the text to: `the installation step`